addons/account/data/mail_template_data.xml

```diff
@@ -8,122 +8,127 @@
         <record id="email_template_edi_invoice" model="mail.template">
             <field name="name">Invoice: Send by email</field>
             <field name="model_id" ref="account.model_account_move"/>
-            <field name="email_from">${(object.invoice_user_id.email_formatted or user.email_formatted) |safe}</field>
-            <field name="partner_to">${object.partner_id.id}</field>
-            <field name="subject">${object.company_id.name} Invoice (Ref ${object.name or 'n/a'})</field>
+            <field name="email_from">{{ (object.invoice_user_id.email_formatted or user.email_formatted) }}</field>
+            <field name="partner_to">{{ object.partner_id.id }}</field>
+            <field name="subject">{{ object.company_id.name }} Invoice (Ref {{ object.name or 'n/a' }})</field>
             <field name="body_html" type="html">
 <div style="margin: 0px; padding: 0px;">
     <p style="margin: 0px; padding: 0px; font-size: 13px;">
         Dear
-        % if object.partner_id.parent_id:
-            ${object.partner_id.name} (${object.partner_id.parent_id.name}),
-        % else:
-            ${object.partner_id.name},
-        % endif
+        <t t-if="object.partner_id.parent_id">
+            <t t-out="object.partner_id.name or ''">Brandon Freeman</t> (<t t-out="object.partner_id.parent_id.name or ''">Azure Interior</t>),
+        </t>
+        <t t-else="">
+            <t t-out="object.partner_id.name or ''">Brandon Freeman</t>,
+        </t>
         <br /><br />
         Here is your
-        % if object.name:
-            invoice <strong>${object.name}</strong>
-        % else:
+        <t t-if="object.name">
+            invoice <strong t-out="object.name or ''">INV/2021/05/0005</strong>
+        </t>
+        <t t-else="">
             invoice
-        %endif
-        % if object.invoice_origin:
-            (with reference: ${object.invoice_origin})
-        % endif
-        amounting in <strong>${format_amount(object.amount_total, object.currency_id)}</strong>
-        from ${object.company_id.name}.
-        % if object.payment_state in ('paid', 'in_payment'):
+        </t>
+        <t t-if="object.invoice_origin">
+            (with reference: <t t-out="object.invoice_origin or ''">SUB003</t>)
```

Comment:
Side dish: why do you have a lot of ``object.char_field or ''`` ? I feel like he should handle Falsy values, allowing to shorten template.